src/services/sessions.js

```diff
@@ -106,7 +106,7 @@ module.exports = class SessionsHelper {
 			// If time slot not available return corresponding error
 			if (timeSlot.isTimeSlotAvailable === false) {
 				const errorMessage = isSessionCreatedByManager
-					? 'INVALID_TIME_SELECTION_FOR_GIVEN_MENTOR'
+					? 'SESSION_CREATION_LIMIT_EXCEDED_FOR_GIVEN_MENTOR'
```

Comment:
Why was this message changed ?

Reply:
@nevil-mathew So we are introducing the new flow where 5 sessions can be created for one mentor for same time . where he can accept one and reject rest .